core/android_make.go

```diff
@@ -72,9 +73,9 @@ func writeIfChanged(filename string, text string) {
 	}
 }
 
-func androidMkWriteString(ctx blueprint.ModuleContext, name string, text string) {
+func androidMkWriteString(sb *strings.Builder, ctx blueprint.ModuleContext, name string) {
```

Comment:
Pref: Can `sb` be the last argument, as `text` was before?

That would follow the reasonably common convention of having the "output" (in this case, `name`, which is used to generate the output filename) earlier than the input (the `strings.Builder`).

It also makes it consistent with `writeIfChanged`.